x/evm/ante/fee.go

```diff
@@ -80,20 +80,9 @@ func (fc EVMFeeCheckDecorator) getMinimumFee(ctx sdk.Context) *big.Int {
 	return fc.evmKeeper.GetMinimumFeePerGas(ctx).RoundInt().BigInt()
 }
 
+// CalculatePriority returns a priority based on the effective gas price of the transaction
 func (fc EVMFeeCheckDecorator) CalculatePriority(ctx sdk.Context, txData ethtx.TxData) *big.Int {
-	if txData.GetGasFeeCap() == nil {
-		return big.NewInt(0)
-	}
-	fee := txData.Fee()
-	tipCapPct := big.NewInt(0)
-	if txData.GetGasTipCap() != nil {
-		tipCapPct = new(big.Int).Quo(txData.GetGasTipCap(), txData.GetGasFeeCap())
-		if tipCapPct.Cmp(big.NewInt(1)) > 0 {
-			tipCapPct = big.NewInt(1)
-		}
-	}
-	discountedFee := new(big.Int).Mul(fee, tipCapPct)
-	adjustedFee := new(big.Int).Quo(discountedFee, state.UseiToSweiMultiplier)
-	nativeGasPrice := new(big.Int).Quo(adjustedFee, new(big.Int).SetUint64(txData.GetGas()))
+	gp := txData.EffectiveGasPrice(big.NewInt(0))
```

Comment:
why are you passing in 0 for base fee?

Reply:
gotcha -- this is bc since base fee is burned and doesnt go to validator, the validator wouldnt care about it for prioritization. Might be good to add a comment here.